lms/static/sass/elements/_progress-circle.scss

```diff
@@ -0,0 +1,59 @@
+.program-progress {
+    width: 300px;
+    margin: 0 auto 30px;
+
+    @media(min-width: $bp-screen-md) {
+        margin-left: 0;
+    }
+}
+
+.progress-heading {
+    color: $blue-d1;
```

Comment:
Good question. My initial thought is the top of this file, so that anyone who brings in the partial gets the variables too. Use ``!default`` so that others can override the values.